backend/typescript/middlewares/validators/authValidators.ts

```
@@ -0,0 +1,17 @@
+import { Request, Response, NextFunction } from "express";
+import isValid from "./isValid";
+
+/* eslint-disable-next-line import/prefer-default-export */
+export const loginRequestValidator = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  if (!isValid(req.body.email, "string")) {
+    return res.sendStatus(400);
```

Can we return an error message please? Same thing for the other validators